### PR TITLE
Swope errata

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: release_notes/errata.adoc [hunk 1]
```diff
@@ -5,6 +5,23 @@ By default, Errata updates are automatically applied when released. See link:../
...
+
+*Note:* {ocp-short} version 4.7 is not supported. You cannot create bare metal managed clusters with the {product-title-short} hub cluster when the hub cluster is hosted on {ocp-short} version 4.7.
+
+* Fixed log errors that occurred when `packageOverrides` was incorrectly specified in a subscription CR. Errors are now logged correctly and the incorrect `packageOverrides` specifications are ignored. (GitHub 10008)
+
+* Updated the list of Azure regions that are available for adding a cluster. https://bugzilla.redhat.com/show_bug.cgi?id=1932430[Bugzilla 1932430]
+
+* Fixed an issue that caused the _Application topology_ page to display an unexpected error. (GitHub 9377)
+
+* Fixed an issue with hub cluster subscription crashing when a Helm subscription is used to subscribe resources from a private Helm channel with only `spec.SecretRef` defined. Now the hub cluster subscription does not crash for this type of Helm subscription. The private Helm repo channel secret must be defined in the same channel namespace. https://bugzilla.redhat.com/show_bug.cgi?id=1932430[Bugzilla 1932430]
```
Comment (dockerymick) [inline release_notes/errata.adoc]: I suggest spelling out repo: repository

**Review by dockerymick — APPROVED** [review]
Made a comment, otherwise this looks good. Thanks Brandi!

File: release_notes/errata.adoc (hunk 1 above)
Comment (swopebe) [reply release_notes/errata.adoc]: thanks!

**Review by blkingrh — APPROVED** [review]
Should "ocp version 4.7 is not supported" be changed to "ocp version 4.7 is not supported on bare metal"? We do support OCP 4.7 with this release with the limitation that it won't work specifically on bare metal.

**Review by cdoan1 — APPROVED** [review]

**Pull request merged**